portfolio/src/main/webapp/index.html

```diff
@@ -7,10 +7,13 @@
     <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
     <script src="https://unpkg.com/material-components-web@latest/dist/material-components-web.min.js"></script>
     <title>MM Portfolio</title>
+    <script src="https://polyfill.io/v3/polyfill.min.js?features=default"></script>
```

Comment:
why is this script needed?

Reply:
not needed right now to my knowledge-updated!